upgrading/topics/migrate_db.adoc

```diff
@@ -39,49 +25,21 @@ By default, this threshold is `300000` records.
 When the number of records is higher than the threshold, the index is not created automatically,
 and there will be a warning message in server logs including SQL commands which can be applied later manually.
 
-To change the threshold, set the `indexCreationThreshold` property, value for the default `connectionsLiquibase` provider:
-
-[source,xml]
-----
-<spi name="connectionsLiquibase">
-    <provider name="default" enabled="true">
-        <properties>
-            <property name="indexCreationThreshold" value="300000"/>
-        </properties>
-    </provider>
-</spi>
-----
-
-Or run this CLI command:
+To change the threshold, set the `index-creation-threshold` property, value for the default `connections-liquibase` provider:
 
 [source,bash]
 ----
-/subsystem=keycloak-server/spi=connectionsLiquibase/:add(default-provider=default)
-/subsystem=keycloak-server/spi=connectionsLiquibase/provider=default/:add(properties={indexCreationThreshold => "300000"},enabled=true)
+kc.[sh|bat] start --spi-connections-liquibase-default-index-creation-threshold=300000
```

Comment:
I think we can remove this. See https://github.com/keycloak/keycloak/issues/9992.

Reply:
Focus here is to remove the WildFly documentation, not to enhance the Quarkus documentation. So this should be handled in a follow-up.